app/assets/stylesheets/components/badges.scss

```diff
@@ -0,0 +1,89 @@
+@import "base/global";
+
+#badge-container {
+	max-width: 700px;
+}
+
+.triangle {
```

Comment:
can you also implement .triangle-left, .triangle-right, .triangle-bottom, and move all this .triangle code into a separate triangle.scss file and do an @include in this?